oketest.sh

```diff
@@ -0,0 +1,194 @@
+    ;;
+    b) availability_domain="${OPTARG}"
+    ;;
+    o) outdir="${OPTARG}"
+    ;;
+    t) test_filter="${OPTARG}"
+    ;;
+    x) threads="${OPTARG}"
+    ;;
+    p) parallel_run="${OPTARG}"
+    ;;
+    d) wdt_download_url="${OPTARG}"
+    ;;
+    i) wit_download_url="${OPTARG}"
+    ;;
+    m) maven_profile_name="${OPTARG}"
+    ;;
+    h) usage 0
+    ;;
+    d) echo "Ignoring -d=${OPTARG}"
+    ;;
+    i) echo "Ignoring -i=${OPTARG}"
+    ;;
+    *) usage 1
+    ;;
+  esac
+done
+
+k8s_version=$(prop 'k8s.version')
+echo "Using Kubernetes version: ${k8s_version}"
+
+
+mkdir -m777 -p "${outdir}"
+export RESULT_ROOT="${outdir}/wl_k8s_test_results"
+if [ -d "${RESULT_ROOT}" ]; then
+  rm -Rf "${RESULT_ROOT}/*"
+else
+  mkdir -m777 "${RESULT_ROOT}"
+fi
+
+echo "Results will be in ${RESULT_ROOT}"
+
+export PV_ROOT="${outdir}/k8s-pvroot"
+if [ -d "${PV_ROOT}" ]; then
+  rm -Rf "${PV_ROOT}/*"
+else
+  mkdir -m777 "${PV_ROOT}"
+fi
+
+echo "Persistent volume files, if any, will be in ${PV_ROOT}"
+
+echo 'Create a OKE cluster'
+mkdir -p "${WORKSPACE}/terraform"
+cp -rf ${terraform_script_dir_name}/*.tf ${WORKSPACE}/terraform/.
+cp -rf ${WORKSPACE}/kubernetes/samples/scripts/terraform/template.tfvars ${WORKSPACE}/terraform/.
+cp -rf ${WORKSPACE}/kubernetes/samples/scripts/terraform/*.sh ${WORKSPACE}/terraform/.
+chmod 777 ${WORKSPACE}/terraform/*.sh
+mkdir -p ${WORKSPACE}/terraform/terraforminstall
+
+
+if ! sh ${WORKSPACE}/terraform/oke.create.sh ${oci_property_file} ${WORKSPACE}/terraform ; then
+sh ${WORKSPACE}/terraform/oke.delete.sh ${oci_property_file} ${WORKSPACE}/terraform
+fi
+
+clusterName=$(prop 'okeclustername')
+
+export KUBECONFIG=${WORKSPACE}/terraform/${clusterName}_kubeconfig
+export PATH=${WORKSPACE}/terraform/terraforminstall:$PATH
+
+echo "creating storage class to setup OFSS ..."
+
+echo "getting MountTarget ID"
+compartment_ocid=$(prop 'compartment.ocid')
+mount_target_id=`oci fs mount-target  list --compartment-id=$compartment_ocid  --display-name=${clusterName}-mt --availability-domain=${availability_domain} | jq -r '.data[] | .id'`
+#export mount_target_id=$mount_target_id
+mt_privateip_id=`oci fs mount-target  list --compartment-id=$compartment_ocid  --display-name=${clusterName}-mt --availability-domain=${availability_domain} | jq -r '.data[] | ."private-ip-ids"[]'`
+
+mt_private_ip=`oci network private-ip get --private-ip-id $mt_privateip_id | jq -r '.data | ."ip-address"'`
+export NFS_SERVER=$mt_private_ip
+echo "Using NFS Server ${NFS_SERVER}"
+
+echo "Creating Storage Class to mount OFSS"
+
+cat << EOF | kubectl apply -f -
+kind: StorageClass
+apiVersion: storage.k8s.io/v1beta1
+metadata:
+  name: oci-fss
+provisioner: oracle.com/oci-fss
+parameters:
+  # Insert mount target from the FSS here
+  mntTargetId: ${mount_target_id}
+EOF
+
+echo 'Set up test running ENVVARs...'
+NODE_IP=`kubectl get nodes -o wide| awk '{print $7}'| tail -n+3`
+if [ -z "$NODE_IP" ]; then
+	echo "retry get node ip ";
```

Comment:
use same indentation in this section.

Reply:
fixed

Reply:
change this indentation to the same as the next line.